### DISCUSSION TITLE
Difference between configuring devtools protocol through "services" and "automationProtocol"

### DISCUSSION THREAD

**Discussion by pillsilly** [webdriverio/webdriverio#8695] [Q&A]
There are two position where to configure `wdio` to use `devtools` protocol:

https://webdriver.io/docs/options/#automationprotocol
and 
https://webdriver.io/docs/devtools-service/#configuration

What would be the difference actually (I tried to read the document but failed to address the difference)?
In my example project both resulted webdriver to use devtools protocol.

**Comment by christian-bromann** [comment]
The `@wdio/devtools-service` is a plugin that offers some devtools related features (see [usage](https://webdriver.io/docs/devtools-service/#usage)). The devtools automation protocol configuration will ensure that automation is run through Chrome DevTools (Puppeteer). You can run the devtools service when automationProtocol is set to `webdriver` as the service has a separate connection to the browser.
